site/docs/guide/introduction.md

```diff
@@ -129,9 +130,49 @@ Please choose what you think is best for you.
 
 ### Prerequisites for Deno
 
-[Install Deno](https://deno.land/#installation) if you have not done that already.
-When you have created your bot, for example in a file called `bot.ts`, you can run it via `deno run --allow-net bot.ts`.
-You can stop it again with `Ctrl+C`.
+Before you can start creating your bot, let's first spend a few minutes on a proper setup for software development.
+This means installing a few tools.
+
+#### Preparing Your Machine for Development
+
+[Install Deno](https://deno.land/manual/getting_started/installation#download-and-install) if you have not done it already.
+
+You also need a text editor that fits well with coding.
+The best one for Deno is Visual Studio Code.
+[Install it.](https://code.visualstudio.com/)
+
+Next, you need to connect VS Code and Deno.
+That's very simple: There is an extension for VS Code that does everything automatically.
+You can install it [as described here](https://marketplace.visualstudio.com/items?itemName=denoland.vscode-deno).
+
+Your system is now ready for bot development! :tada:
+
+#### Developing a Bot
+
+Create a new directory somewhere.
+It will contain your bot project.
+Open this new directory in VS Code.
+
+```sh
+mkdir ./my-bot
+cd ./my-bot
+code .
+```
+
+> If you’re on macOS and the `code` command is not available, just open VS Code, hit `Ctrl+Shift+P`, type "shell command", and hit Enter.
```

Comment:
I'm not so familiar with macOS. I believe the CTRL key in windows is CMD in macOS?
```suggestion
> If you’re on macOS and the `code` command is not available, just open VS Code, hit `Cmd+Shift+P`, type "shell command", and hit Enter.
```

Reply:
This one is for @roj1512 to decide

Reply:
Thanks for correcting me, @CikiMomogi. I’d even put the command icon.

Reply:
Btw, we can use the kbd tag to make them look more stylish.

![image](https://user-images.githubusercontent.com/74030149/218291293-aa5aeb82-36f2-49dc-a2c8-7d16f33425b6.png)